modules/doc/content/newsletter/2022_04.md

```diff
@@ -5,6 +5,19 @@ This MOOSE Newsletter edition is in progress. Please check back in May 2022
 for a complete description of all MOOSE changes.
 !alert-end!
 
+## MOOSE Supports Scientific Machine Learning using C++ API of Pytorch
+
+MOOSE can now be configured to compile with libtorch (C++ back end of Pytorch). This
+allows the utilization of neural networks for scientific machine learning
+directly in MOOSE and consequently in MOOSE-based applications.
+For the installation istructions, click [here](getting_started/installation/install_libtorch.md).
+More information on the utilization of artificial neural networks can be found here:
+[LibtorchArtificialNeuralNet.md].
+
+Known limitations of the MOOSE-libtorch coupling:
+- It only supports serial neural net training
+- It only support CPU-based training
```

Comment:
The MOOSE-LibTorch coupling only supports serial neural net training and does not support GPU-based training.